scripts/db_migrator.py

```diff
@@ -174,6 +174,22 @@ def migrate_copp_table(self):
         for copp_key in keys:
             self.appDB.delete(self.appDB.APPL_DB, copp_key)
 
+    def migrate_feature_table(self):
+        '''
+        Combine CONTAINER_FEATURE and FEATURE tables into FEATURE table.
+        '''
+
+        feature_table = self.configDB.get_table('FEATURE')
+        for feature, config in feature_table.items():
+            state = config.pop('status', 'disabled')
+            config['state'] = state
+            self.configDB.set_entry('FEATURE', feature, config)
```

Comment:
Should this be `mod_entry` too?

Reply:
mod_entry will leave "status" but we need "state"